src/airport.js

```diff
@@ -29,6 +30,15 @@ class Airport {
       console.log(`the Plane took off`);
   };
   checkPlane = (plane) => this.airportPlanes.includes(plane);
+
+  checkWeather = () => {
```

Comment:
i was not sure if i should have set the  weather from the beginning as a condition for the plane to take off or to land. I chose to do it step by step as required.
i'm not sure if this is how i should have set the weather , i google it as i didn't know how to do it.